### PR TITLE
Adds some new tips for fighting the clockcult

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
tweak: Added some new tips to the list for fighting the clockcult!
/:cl:

Apparently a lot of this stuff is either not well-known or not utilized, so hopefully this will improve counterplay a bit. I pointed out the holy water spray, the null rods' power against marauders, and blessing water tanks.

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -163,6 +163,9 @@ As a Servant, declaring war empowers a huge amount of your tools and constructs,
 As a Servant, converting or sabotaging Science and Genetics can make defending the Ark much easier.
 As a Servant, the Clockwork Arnaments scripture allows you to summon armor and/or a weapon at will. Use it whenever you unlock it!
```
Comment (ChangelingRain) [inline strings/tips.txt]: arnaments

Comment (Xhuis) [reply strings/tips.txt]: The name is already capitalized and spelled correctly...?

Comment (Anonmare) [reply strings/tips.txt]: >Arnaments

You mean, armaments?

Comment (Xhuis) [reply strings/tips.txt]: Oh, bother!

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

**Pull request merged**